General/User settings docu review (#1818)
SAP/luigi: docs/general-settings.md

<!-- add-attribute:class:success -->
>**TIP:** For learning and testing purposes, use the [Luigi Fiddle](https://fiddle.luigi-project.io) page where you can configure a sample Luigi application.

> The configuration file contains a section called **Settings** in which you can configure additional Luigi options. This is an example of how settings parameters can be used:

Reviewer: ```suggestion
The configuration file contains a section called **Settings** where you can configure additional Luigi options. This is an example of how settings parameters can be used:
```

}
```

## Parameters

> Below is a list of the parameters you can use in the `settings:` Luigi configuration section.

Reviewer: ```suggestion
Below a list of the parameters you can use in the `settings:` Luigi configuration section.
```

Author: In this case I think it's best to leave the "is", as it's the only verb in the sentence and removing it would make it incomplete: https://examples.yourdictionary.com/reference/examples/examples-of-complete-sentences.html

### User settings

> Luigi allows you to display a user settings dialog and manage user data. In order to do that, it is neccessary to define a user settings schema. The schema is defined in a `userSettingGroups` object.

Reviewer: ```suggestion
Luigi allows you to display a user settings dialog and to manage user data, through defining a user settings schema. The schema is defined in a `userSettingGroups` object.
```
